include/bouncer.h

```diff
@@ -66,13 +66,13 @@ enum SocketState {
 	SV_ACTIVE,		/* pool->active_server_list */
 	SV_ACTIVE_CANCEL,	/* pool->active_cancel_server_list */
 	SV_USED,		/* pool->used_server_list */
-	SV_TESTED		/* pool->tested_server_list */
+	SV_TESTED,	/* pool->tested_server_list */
```

Comment:
I personally stopped caring about pretty much all aspects of codestyling. As long as it's consistent and automatically enforced I'm happy. 

The only exception to that is trailing/dangling commas. I think those should be used whenever the language allows them to be (when items are on multiple lines). They keep future diffs clean, and allows reordering of lines without problems. This blog outlines my viewpoint on this pretty well:  https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8

Reply:
I do like this change, but maybe it could be put into a separate commit? The first commit should just normalize the existing formatting, then we can evolve it.

Reply:
Removed for now. Once this initial one is merged I'll create a few PRs dedicated to some possible styling changes (this will be the first one).